docs/docs/extend/plugins/panel.md

```diff
@@ -160,3 +161,93 @@ Please have a look at the css files for more options. The last line renders the
 
 !!! tip "Give it a try"
     Each time you press the button, the value will be increased.
+
+### Handling user input
+
+A common user case is user input that needs to be passed from the panel into
+the plugin for further processing. Lets have a look at another example. We
+will define two user input fields. One is an integer the other one a string.
+A button will be defined to submit the data. Something like that:
+
+{% with id="panel_with_userinput", url="plugin/panel_with_userinput.png", description="Panel with user input" %} {% include "img.html" %} {% endwith %}
+
+Here is the plugin code:
+
+```python
+from django.urls import path
+from django.http import HttpResponse
+
+from plugin import InvenTreePlugin
+from plugin.mixins import PanelMixin, UrlsMixin
+
+class ExamplePanel(PanelMixin, InvenTreePlugin, UrlsMixin):
+
+    NAME = "ExamplePanel"
+    SLUG = "examplepanel"
+    TITLE = "Example for data input"
+    AUTHOR = "Michael"
+    DESCRIPTION = "This plugin passes user input from the panel to the plugin"
+
+# Create the panel that will display on every view
+    def get_custom_panels(self, view, request):
+        panels = []
+        panels.append({
+            'title': 'Example Info',
+            'icon': 'fa-industry',
+            'content_template': 'example_panel/example.html',
+        })
+        return panels
+
+    def setup_urls(self):
+        return [
+                path("example/<int:layer>/<path:size>/",
+                    self.SelectCompany, name = 'transfer'),
+        ]
+
+# Define the function that will be called.
+    def SelectCompany(self, request, layer, size):
```

Comment:
Following python conventions, variables and function names should be named in lower_snake_case. PascalCase is used for class names.